test_scripts/API/VehicleData/GearStatus/026_Resumption_of_subscription_for_2_apps_to_the_gearStatus_param_after_IGN_OFF_ON_cycle.lua

```diff
@@ -0,0 +1,59 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0266-New-vehicle-data-GearStatus.md
+-- Description: SDL resumes the subscription for 'gearStatus' parameter for two Apps after IGN_OFF/ON.
+-- Precondition:
+-- 1) Two apps are registered and activated.
+-- 2) Apps are subscribed to `gearStatus` data.
+-- 3) IGN_OFF/IGN_ON cycle is performed.
+-- In case:
+-- 1) Mobile app1 and app2 register with actual hashID.
+-- SDL does:
+--  a) start data resumption for both apps.
+--  b) resume the subscription and sends VI.SubscribeVD request to HMI.
+--  c) after success response from HMI SDL resumes the subscription.
+-- 2) HMI sends OnVD notification with subscribed VD.
+-- SDL does:
+--  a) resend OnVD notification to appropriate mobile apps.
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/API/VehicleData/GearStatus/common')
+
+--[[ Local Variables ]]
+local appId1 = 1
+local appId2 = 2
+local isExpected = true
```

Comment:
Value can be defined for 2 functions like
```suggestion
local isExpectedSubscribeVDonHMI = true
```